mappings/net/minecraft/client/particle/TechnicalBlockParticle.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_6237 net/minecraft/client/particle/TechnicalBlockParticle
```

Comment:
Duplicate of https://github.com/FabricMC/yarn/pull/2271/files#diff-0cead92be21cba427351a9b67cde0a961202ea7ed9e841618d65cb34d3c34ea6

Reply:
This shouldn't have been commited. >.>